src/ray/util/util.h

```diff
@@ -382,4 +382,16 @@ void QuickExit();
 /// \return the foramtted value
 std::string FormatFloat(float value, int32_t precision);
 
+// Insert default labels to node labels.
+absl::flat_hash_map<std::string, std::string> GetFinalNodeLabels(
+    const absl::flat_hash_map<std::string, std::string> &labels,
+    const std::string &node_id,
+    const std::string &node_ip);
+
+#define RAY_LABEL_KEY_PREFIX "ray.io/"
+/// Default node label key: node_id
+constexpr char kLabelKeyNodeID[] = RAY_LABEL_KEY_PREFIX "node_id";
```

Comment:
constant should be defined in `constants.h` file.

Reply:
Fixed.